tools/lxdclient/client.go

```diff
@@ -271,7 +271,7 @@ func verifyDefaultProfileBridgeConfig(client *lxd.Client, networkAPISupported bo
 		return "", errors.Trace(err)
 	}
 
-	// If the default profile doesn't have eth0 in it, then the user has messed
+	// If the default profile has eth0 in it, then the user has messed
```

Comment:
Checking for eth0 is also a poor way of checking to see if the network config has been changed. We need a better way of communicating default/custom.